nixos/modules/tasks/network-interfaces-scripted.nix

```
@@ -253,7 +253,11 @@ let
             '';
           };
 
-        createTunDevice = i: nameValuePair "${i.name}-netdev"
+          createTunDevice = i: nameValuePair "${i.name}-netdev"
```

```suggestion
        createTunDevice = i: nameValuePair "${i.name}-netdev"
```
did your editor indent this by mistake?

---

Whops. Yeah my editor added the indent.  Removed.